docs/08-developer-guide/02-run/02-docker-compose.md

```diff
@@ -10,3 +10,143 @@ sidebar_label: Docker Compose
 - Define the Ozone runner image at `ozone-docker-runner` and its purpose to wrap Ozone binaries.
 - How to define which image gets used in the cluster (runner or other pre-built image).
 - Changing configurations in Docker Compose (`docker-config` file, transformation.py, etc).
+
+This guide explains how to run Apache Ozone using Docker Compose, either with locally built sources or pre-built images.
+
+## Prerequisites
+
+- Docker Engine 20.10.0 or higher
+- Docker Compose V2
```

Comment:
Let's link to the relevant pages on the docker website for installing these components.

Reply:
Fixed